.eslintrc.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
+    "env": {
+        "browser": true
+    },
+    "extends": "eslint:recommended",
+    "globals": {
+        "google": true,
+        "result": true,
+        "module": true,
+        "degreesToradians": true,
+        "getRadiusLatlng": true
```

Comment:
This should not be added to globals. You were getting a linting error pointing to it because it's not spelled correctly. Your module is now returning something that's not actually defined anywhere.